tests/generation_tests.py

```diff
@@ -32,3 +34,45 @@ def test_grouped():
 def test_resolution():
     options = get_clean_opts(["--tab-artwork-resolution=300"])
     main.generate(options)
+
+
+def test_no_group_global():
+    options = get_clean_opts([])
+    assert not options.group_global
+
+
+def test_group_global():
+    options = get_clean_opts(["--group-global"])
+    expected_groups = [
+        "prophecies",
+        "traits",
+        "hexes",
+        "events",
+        "landmarks",
+        "state",
+        "trait",
+        "way",
+        "prophecy",
+        "boons",
+        "projects",
+        "event",
+        "boon",
+        "landmark",
+        "ally",
+        "allies",
```

Comment:
Why are both of these here? The test passes, so this is what the code is doing, but this doesn't seem right to me.

Reply:
hmm, actually, they're all duplicated, it's just not clear because this list is in whatever order it was in when I copied it from my debugger, not alphabetized. I'll clean that up, at least.